pkg/resources/mutation/cluster/provisioningcluster.go

```diff
@@ -2,44 +2,91 @@ package cluster
 
 import (
 	"fmt"
+	"reflect"
+	"strings"
 
+	"golang.org/x/exp/slices"
+
+	"github.com/blang/semver"
+	v1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
 	"github.com/rancher/webhook/pkg/admission"
+	"github.com/rancher/webhook/pkg/auth"
+	"github.com/rancher/webhook/pkg/clients"
+	v3 "github.com/rancher/webhook/pkg/generated/controllers/management.cattle.io/v3"
 	objectsv1 "github.com/rancher/webhook/pkg/generated/objects/provisioning.cattle.io/v1"
-	"github.com/rancher/webhook/pkg/resources/mutation"
+	"github.com/rancher/webhook/pkg/patch"
+	psa "github.com/rancher/webhook/pkg/podsecurityadmission"
+	"github.com/rancher/wrangler/pkg/data/convert"
+	corecontroller "github.com/rancher/wrangler/pkg/generated/controllers/core/v1"
+	"github.com/sirupsen/logrus"
 	admissionv1 "k8s.io/api/admission/v1"
 	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/utils/trace"
 )
 
+const (
+	KubeAPIAdmissionConfigOption = "admission-control-config-file"
```

Comment:
Exported constants should have a comment

Reply:
fixed.